electronicparsers/wannier90/parser.py

```diff
@@ -444,20 +501,92 @@ def parse_hoppings(self):
                 self.logger.warning(
                     'Could not parse the hopping matrix values. Please, revise your output files.'
                 )
+    
+    def parse_tb_bands(self):
+        hr_files = get_files('*hr.dat', self.filepath, self.mainfile)
+        if not hr_files:
+            return
+        if len(hr_files) > 1:
+            self.logger.warning('Multiple hopping data files found.')
```

Comment:
Also state what the next step / decision is.

```suggestion
            self.logger.warning('Multiple hopping data files found. Picking the first file found.')
```